orange/src/main/java/com/hartwig/hmftools/orange/report/tables/HLAAlleleTable.java

```diff
@@ -51,14 +52,26 @@ public static Table build(@NotNull String title, float width, @NotNull List<Lila
     private static List<LilacAllele> sort(@NotNull List<LilacAllele> alleles)
     {
         return alleles.stream()
-                .sorted(Comparator.comparing(LilacAllele::allele).thenComparingInt(LilacAllele::refFragments))
+                .sorted(Comparator.comparing(LilacAllele::allele)
+                        .thenComparingInt(allele -> allele.refFragments() != null ? allele.refFragments() : 0))
                 .collect(Collectors.toList());
     }
 
     @NotNull
-    private static String fragmentString(int fragments, boolean isTumorFail)
+    private static String fragmentString(@Nullable Integer fragments, boolean isTumorFail)
     {
-        return !isTumorFail ? String.valueOf(fragments) : ReportResources.NOT_AVAILABLE;
+        if(fragments == null)
+        {
+            return ReportResources.NONE;
```

Comment:
I'd think NOT_AVAILABLE would be better than NONE? 

I am fine if you want to stick to NONE but that term sounds a bit too similar to 0 which is also wrong.